File: Packets/DisconnectResponse.vb
Public Class DisconnectResponse
    Inherits Packet

    Public Overrides Property OPCode As Byte = &H2
    Public Property Unknown As Int32 = 9001

    Sub New()
        MyBase.New(Nothing)
    End Sub

    Public Overrides Function GetByteArray() As Byte()

    End Function
End Class

File: Packets/DisconnectResponse.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
public class DisconnectResponse : Packet
{

	public override byte OPCode { get; set; }
	public int Unknown { get; set; }

	public DisconnectResponse() : base()
	{
	}

	public override byte[] GetByteArray()
	{
		Payload = BitConverter.GetBytes(Unknown);
		return Package();
	}
}
